niceman/distributions/tests/test_debian.py

```diff
@@ -136,4 +140,47 @@ def _run_dpkg_query(subfiles):
         '/usr/bin/fail2ban-server': {'name': u'fail2ban'},
         '/usr/lib/afni/bin/afni': {'name': u'afni'},
         '/bin/sh': {'name': u'dash'}
-    }
\ No newline at end of file
+    }
+
+@pytest.fixture
+def setup_packages():
+    """set up the package comparison tests"""
+    a = DEBPackage(name='p1')
+    b = DEBPackage(name='p1', version='1.0')
+    c = DEBPackage(name='p1', version='1.1')
+    d = DEBPackage(name='p2')
+    return (a, b, c, d)
+
+def test_package_satisfies(setup_packages):
+    (p1, p1v10, p1v11, p2) = setup_packages
+    assert p1.satisfies(p1) is True
```

Comment:
sufficient to
```python
assert p1.satisfies(p1)
assert not p1.satisfies(p1v10)
```

Reply:
I'd set it up this way so I could return None and make all tests fail to start with.  You're right, at this point this idiom isn't needed any more.